XenAdmin/Dialogs/HealthCheck/HealthCheckSettingsDialog.cs

```diff
@@ -316,7 +319,6 @@ private void credentials_TextChanged(object sender, EventArgs e)
 
         private void xsCredentials_TextChanged(object sender, EventArgs e)
         {
-            UpdateButtons();

```

Comment:
Yes this was intentional, to only allow the user to click ok after the credentials have been checked..since we couldn't do the check when the ok button is pressed